.github/workflows/iroha2-base.yml

```diff
@@ -34,5 +34,6 @@ jobs:
         with:
```

Comment:
(Line: 19, Col: 9): Unrecognized named-value: 'matrix'. Located at position 1 within expression: matrix.tag
We can try `if: github.ref == '${{ github.ref_name }}'`

Reply:
No, `github.ref_name` doesn't manage branch context and causes base image building for three tags in parallel.

Reply:
Yeah, that's exactly what I'm fixing in aa6c29c23d4110c9c6e9d89fc37cb7a15dc06b88 after https://github.com/hyperledger/iroha/pull/2662#discussion_r956946372